shell/platform/tizen/external_texture_pixel_egl.cc

```diff
@@ -0,0 +1,70 @@
+// Copyright 2020 Samsung Electronics Co., Ltd. All rights reserved.
```

Comment:
The GL header used for external_texture_pixel_egl/evasgl is different. I split the class in two because this is determined by the renderer_type of the runtime. I think we can abstract it to the external_texture_pixel class to avoid code duplication, But if I have to, I want to do(refactoring) it later after this PR is merged.

If external_texture_pixel is a feature we don't need, please tell me and I'll remove it

Reply:
The related part will be refactored soon (https://github.com/flutter-tizen/engine/issues/285) so it would be better to make only a minimal change. As far as I can see the only difference between the two files is the included headers, and there's no difference in the actual implementations, so it shouldn't be necessary to split the files.

Reply:
Oops. I thought we could use `#ifdef` for the includes depending on the current profile but it doesn't seem to be a feasible solution. Is there any other way to unsplit the files?

Reply:
>  I think we can abstract it to the external_texture_pixel class to avoid code duplication, But if I have to, I want to do(refactoring) it later after this PR is merged.

I don't know think of any special way. um...  I don't know much about GpuBufferTexture, but if GpuBufferTexture has to be divided into EvasGL, and EGL, I don't think it's much different from now.

Reply:
We decided to remove the pixel type texture support after offline discussion.

Reply:
I removed the pixel type texture classes